src/Illuminate/Database/Schema/Blueprint.php

```diff
@@ -731,9 +731,32 @@ public function foreign($columns, $name = null)
      */
     public function id($column = 'id')
     {
+        if (Builder::$defaultIdType === 'uuid') {
+           return $this->uuid($column)->primary();
+        } elseif (Builder::$defaultIdType === 'ulid') {
+          return  $this->ulid($column)->primary();
+        }
         return $this->bigIncrements($column);
```

Comment:
You could add some error handling here
```suggestion
        return match (Builder::$defaultIdType) {
            'uuid' => $this->uuid($column)->primary(),
            'ulid' => $this->ulid($column)->primary(),
            'id' => $this->bigIncrements($column),
            default => throw new InvalidArgumentException("Key type must be 'int', 'uuid', or 'ulid'."),
        };
```